tests/core/metrics/aggregateMetrics.test.ts

```diff
@@ -0,0 +1,57 @@
+import { describe, expect, it } from 'vitest';
+import type { FileMetrics } from '../../../lib/core/metrics/calculateIndividualFileMetrics.js';
```

Comment:
This line should be:
```typescript
import type { FileMetrics } from '../../../src/core/metrics/calculateIndividualFileMetrics.js';
```

Reply:
Thanks for catching that. Not sure how it was passing here, but yeah, certainly breaking main.

Here's the fix:
https://github.com/yamadashy/repomix/pull/218